File: code-coaches/medium/pig-latin/pig_latin.py
def to_pig_latin(sentence: str) -> str:
    words = sentence.split()
    pig_latin_words = []
    for word in words:
        pig_latin_word = word[1:] + word[0] + "ay"
        pig_latin_words.append(pig_latin_word)

    return " ".join(pig_latin_words)


def main():
    sentence = input()
    print(to_pig_latin(sentence))


if __name__ == "__main__":
    main()

File: code-coaches/medium/pig-latin/pig_latin.rb
def to_pig_latin(sentence)
  words = sentence.split(" ")
  pig_latin_words = words.map do |word|
    first_letter = word[0]
    rest_of_word = word[1..-1]
    "#{rest_of_word}#{first_letter}ay"
  end

  return pig_latin_words.join(" ")
end

def main
  sentence = gets.chomp
  puts to_pig_latin(sentence)
end

if __FILE__ == $0
  main
end